src/app/server/Server.cpp

```diff
@@ -255,11 +255,11 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
     // This code is necessary to restart listening to existing groups after a reboot
     // Each manufacturer needs to validate that they can rejoin groups by placing this code at the appropriate location for them
     //
-    // This is disabled for thread device because the same code is already present for thread devices in
+    // This is disabled for Thread/LWIP devices using dedicated Inet endpoint implementations, because they run the same code in
     // src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread_LwIP.cpp
-#if !CHIP_DEVICE_CONFIG_ENABLE_THREAD
+#if !CHIP_SYSTEM_CONFIG_USE_OPEN_THREAD_ENDPOINT
```

Comment:
Even with the updated comment, it is confusing why this #define targets the correct platforms.

Reply:
Rephrased the comment.

Reply:
Thanks.